include/libdnf-cli/exception.hpp

```diff
@@ -88,6 +88,29 @@ class CommandExitError : public Error {
     int exit_code;
 };
 
+/// Exception used when non-standard exit code should be returned
```

Comment:
Maybe just improve the doc string as it is the same as with `CommandExitError` - add an explanation what the `Silent` prefix means...

Reply:
Ah, meant to change this, thank you.